packages/ws/src/utils/constants.ts

```diff
@@ -0,0 +1,22 @@
+import { REST } from '@discordjs/rest';
+import { APIVersion } from 'discord-api-types/v10';
+import type { WebSocketManagerOptions } from '../struct/WebSocketManager';
+
+/**
+ * Default options used by the manager
+ */
+export const DefaultWebSocketManagerOptions: WebSocketManagerOptions = {
+	shardCount: null,
+	shardIds: null,
+	get rest() {
```

Comment:
Does this work o.o

Reply:
Yeah, I took inspiration from REST default optioks - see the `agent` prop.